packages/components/package.json

```diff
@@ -54,45 +54,43 @@
     "clsx": "^1.1.1"
   },
   "devDependencies": {
-    "@babel/cli": "^7.14.8",
-    "@babel/core": "^7.15.0",
-    "@babel/plugin-transform-modules-commonjs": "^7.15.0",
-    "@babel/preset-env": "^7.15.0",
+    "@babel/cli": "^7.15.7",
+    "@babel/core": "^7.15.8",
+    "@babel/plugin-transform-modules-commonjs": "^7.15.4",
+    "@babel/preset-env": "^7.15.8",
     "@babel/preset-react": "^7.14.5",
     "@babel/preset-typescript": "^7.15.0",
-    "@chanzuckerberg/axe-storybook-testing": "^4.1.2",
+    "@chanzuckerberg/axe-storybook-testing": "^4.1.3",
     "@chanzuckerberg/story-utils": "^2.1.0",
     "@percy/storybook": "^3.3.1",
-    "@storybook/addon-a11y": "^6.3.8",
-    "@storybook/addon-essentials": "^6.3.8",
+    "@storybook/addon-a11y": "^6.3.12",
+    "@storybook/addon-essentials": "^6.3.12",
     "@storybook/addon-postcss": "^2.0.0",
-    "@storybook/addons": "^6.3.8",
```

Comment:
@storybook/addon seems to be used when creating addons, not when actually using storybook